askama/src/lib.rs

```diff
@@ -59,17 +59,23 @@
 //!   in the configuration file. The default syntax , "default",  is the one
 //!   provided by Askama.
 
-#![allow(unused_imports)]
```

Comment:
I forget why we have `unused_imports` here, if we can get rid of it that certainly seems like a win.